.github/workflows/release-azure-pipelines.yml

```diff
@@ -267,18 +274,24 @@ extends:
                 artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}-signed
                 targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}-signed
             outputs:
-            - ${{ each config in parameters.buildConfigs }}:
-              - output: pipelineArtifact
-                artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
-                targetPath: $(Build.SourcesDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
+            - output: pipelineArtifact
+              artifactName: azureauth-${{ parameters.version }}-packaged
+              targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-packaged
           steps:
+          - task: PowerShell@2

```

Comment:
We didn't have this step for signing. So does ESRP task creates the `$targetPath` folder if it doesn't exist and in the case of packaging, we need to explicitly create the `$targetPath` folder?

Reply:
Right, also in case of packaging, 1ES PT official version only allows us to upload folder artifacts and not packaged files like zip,tar. So, we had to create a `packaged` folder and put all the files there to upload it for release stage . 😅